src/pages/registerForm.js

```diff
@@ -0,0 +1,110 @@
+import { Form, Input, Button} from 'antd';
+import { Link, useNavigate } from 'react-router-dom';
+import { routes } from '../config/routes';
+import { httpsService } from '../utils/https.service';
+import '../styles/registerPage.scss';
+
+const formItemLayout = {
+  labelCol: {
+    xs: {
+      span: 24,
+    },
+    sm: {
+      span: 8,
+    },
+  },
+  wrapperCol: {
+    xs: {
+      span: 24,
+    },
+    sm: {
+      span: 16,
+    },
+  },
+};
+const tailFormItemLayout = {
+  wrapperCol: {
+    xs: {
+      span: 24,
+      offset: 0,
+    },
+    sm: {
+      span: 16,
+      offset: 8,
+    },
+  },
+};
+
+export const RegisterForm = () => {
+  const [form] = Form.useForm();
+  const navigate = useNavigate();
+  
+
+  const handleRegister = async (body) => {
+      
+    //console.log(body);
```

Comment:
remove unused line and console logs and comments. Please install eslint and prettier into project